src/module_TOUCHSCREEN/TouchscreenApp.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Salt Lake Community College
+ * Spring 2017 Semester
+ * CSIS-2810 Computer Architecture
+ * Instructor: Jon McGowan
```

Comment:
I have hesitations about this particular header on each class file. Were this an "assignment" then I wouldn't mind it but in a project like this I'd probably suggest we remove it or put in a header specific to this project and have our names on it. Lets discuss this on Monday. I won't have this stop the pull request.